examples/demo-stf/src/tests/mod.rs

```diff
@@ -60,3 +30,9 @@ pub fn has_tx_events(apply_blob_outcome: &BatchReceipt<SequencerOutcome, TxEffec
 
     events.peekable().peek().is_some()
 }
+
+pub fn new_test_blob(batch: Batch, address: &[u8]) -> TestBlob {
+    let address = Address::try_from(address).unwrap();
+    let data = batch.try_to_vec().unwrap();
+    TestBlob::new(data, address, [0; 32])
+}
```

Comment:
This looks like it's duplicated in sov-cli

Reply:
Moved to https://github.com/Sovereign-Labs/sovereign-sdk/issues/431